src/lexer.coffee

```diff
@@ -116,10 +117,18 @@ exports.Lexer = class Lexer
     if id is 'from' and @tag() is 'YIELD'
       @token 'FROM', id
       return id.length
-    if id is 'as' and @seenImport and (@tag() is 'IDENTIFIER' or @value() is '*')
-      @tokens[@tokens.length - 1][0] = 'IMPORT_ALL' if @value() is '*'
-      @token 'AS', id
-      return id.length
+    if id is 'as' and @seenImport
+      # loop hack
```

Comment:
Are you sure about that?
```
coffee -ne "import { as as as } from 'lib'"
Block
  ImportDeclaration
    ImportClause
      ImportSpecifierList
        ImportSpecifier IdentifierLiteral: as IdentifierLiteral: as StringLiteral: 'lib'
```
See https://github.com/jashkenas/coffeescript/blob/master/test/modules.coffee#L587-L609